templates/printer-added.tmpl

```diff
@@ -1,4 +1,13 @@
-<H2 CLASS="title">Add Printer</H2>
+<H2 CLASS="title">Add Printer {printer_name}</H2>
+
+<P>Printer drivers and raw queues are deprecated and will stop working in a future version of CUPS.
```

Comment:
Maybe move this after the "added successfully" message, and put it in a blockquote, e.g.:

```
<blockquote><b>Note:<b> Printer drivers ...</blockquote>
```

Reply:
Ack, I'll change it.